index.html

```diff
@@ -137,12 +137,32 @@
     </div>
   </script>
 
+  <!-- Project List item -->
+  <script type="text/template" id="project-list-template">
+    <%
+      var classNames = [
+        "project-list",
+        starred ? "starred" : "",
+        featured ? "featured" : "",
```

Comment:
we don't need to know whether the project is "featured" or not since we're not doing any special treatment to those featured ones on the Favs view